pkg/core/consensus/blockgenerator/candidate/blockgenerator.go

```diff
@@ -139,7 +139,7 @@ func (bg *generator) execute(ctx context.Context, txs []transactions.ContractCal
 // fetchOrTimeout will keep trying to FetchMempoolTxs() until either
 // we get some txs or or the timeout expires.
 func (bg *generator) fetchOrTimeout(keys [][]byte) ([]transactions.ContractCall, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
```

Comment:
The timeout for the selection phase was previously around 5-6 seconds. We should not go with 15 seconds for selection as it will lead to very slow block time in a full network

Reply:
This change came from rebasing with `master`. It's been merged in `master`. Please address this with @herr-seppia 

From master
https://github.com/dusk-network/dusk-blockchain/blob/f9af476da7099f02586507dfaec1d8276072e5c4/pkg/core/consensus/blockgenerator/candidate/blockgenerator.go#L142